app/services/ui_notifications/hosts/provision.rb

```diff
@@ -0,0 +1,51 @@
+module UiNotifications
+  module Hosts
+    class Provision < Base
+      def deliver!
+        case state
+        when :build_completed
+          host_build
+        end
+      end
+
+      private
+
+      def host_build
+        logger.debug('Host finished provisioning (build is done) - checking for notficiations')
+        blueprint = host_build_blueprint
+
+        if blueprint.notifications.where(:subject => subject).empty?
+          logger.debug('creating new UI notification host_build_completed')
+          ::Notification.create!(
+            initiator: User.find(subject.owner_id),
```

Comment:
`initiator` has to be present, but `subject.owner_id` doesn't... so I don't believe it's a good choice here